controllers/cloud.redhat.com/pool_controller.go

```diff
@@ -71,7 +71,7 @@ func (r *NamespacePoolReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	pool.Status.Creating = status["creating"]
 
 	for i := r.underManaged(pool); i > 0; i-- {
-		start_create_ns_timer := time.Now()
+		startCreateNamespaceTimer := time.Now()
```

Comment:
If you're going to modify this - you may want to do something similar to the other one where we use teh creation timestamp of the namespace to get an accurate time of the set up.